Source/WebCore/editing/cocoa/DataDetection.mm

```diff
@@ -647,7 +686,7 @@ static inline CFComparisonResult queryOffsetCompare(DDQueryOffset o1, DDQueryOff
 
             lastNodeContent = currentTextNode->data().substring(range.end.offset, currentTextNode->length() - range.end.offset);
             lastTextNodeToUpdate = WTFMove(currentTextNode);
-        }        
```

Comment:
Would be really nice if you could fix your IDE settings to stop doing this. So much noise in the PR because of this.